README.md

```diff
@@ -183,11 +183,6 @@ make install-poetry
 make install-env
 ```
 
-* Now you can use this venv for development. You can activate it in your shell by running:
-
-```bash
-make activate-env # or simply, poetry shell
```

Comment:
1. `activate-env` is not defined in the Makefile
2. poetry shell is not needed, because of virtual environment

Reply:
You're right. Thanks for the contribution